docker_mcp/models/CLAUDE.md

```
@@ -1,29 +1,151 @@
 # Models Layer - Development Memory
 
-## Pydantic Model Architecture
+## Pydantic v2 Model Architecture (2.0+)
 
-### Base Model Pattern
+### Modern Base Model Pattern
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)